koiki/tests/test_client.py

```diff
@@ -59,16 +57,30 @@ def setUp(self):
             ],
         }
 
+        responses.add(responses.GET, 'https://wcfmmp_testing_host/wp-json/wcfmmp/v1/settings/id/6',
+                      status=200,
+                      json={
+                        "store_email": "queviure@lazona.coop",
+                        "phone": "",
+                        "address": {
+                            "street_1": "",
+                            "street_2": "",
+                            "city": "",
+                            "zip": "",
+                            "country": "ES",
+                            "state": ""
+                        }
+                      })
```

Comment:
I tried to mock CreateDelivery stubbing url() and body() in order to decouple this Koiki's module tests from the requests to WP but didn't succeed. We'll get to it later on. I think it's important we define a clear boundary between `koiki`, `api` and `woocommerce` and it's already becoming a bit of a mess already.